packages/calling/src/common/types.ts

```diff
@@ -231,24 +231,24 @@ interface Manager {
 }
 
 interface EnterpriseUser {
-  department: string;
-  manager: Manager;
+  department?: string;
+  manager?: Manager;
 }
 
 interface Resource {
   schemas: string[];
   id: string;
   userName: string;
-  active: boolean;
-  name: Name;
-  displayName: string;
-  emails: URIAddress[];
+  active?: boolean;
+  name?: Name;
+  displayName?: string;
+  emails?: URIAddress[];
   userType: string;
```

Comment:
Also, will this not be a build issue? I thought optionals are usually kept at the end of the method signature and not before a mandatory arguement..